apps/condo/featureflags/featureToggleManager.js

```diff
@@ -0,0 +1,36 @@
+const { GrowthBook } = require('@growthbook/growthbook')
```

Comment:
Is it a backend? Probably it’s a common / util

Reply:
moved featureflags folder to packages